elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -311,34 +287,29 @@ public String getNameFromAlias(Object entity, String alias) {
     }
 
     /**
-     * Initialize an entity.
+     * Initialize an entity
```

Comment:
I see all of these comments changed; I believe your and Dan's IDE's may be set differently :( iirc Dan recently "fixed" (this is a matter of style, I suppose) all comments to end with `.`

@danchen thoughts/comments?